cmd/state/internal/cmdtree/cmdtree.go

```diff
@@ -223,6 +224,12 @@ func newStateCommand(globals *globalOptions, prime *primer.Values) *captain.Comm
 				Description: locale.T("flag_state_version_description"),
 				Value:       &opts.Version,
 			},
+			{
+				Name:      "no-interactive",
```

Comment:
```suggestion
				Name:      "non-interactive",
```

My bad, I typo'd this in the ACs.